.github/actions/go-test/entrypoint.sh

```diff
@@ -19,27 +19,53 @@
 set -e
 
 download_go() {
+	. build/functions.sh
+	if verify_and_set_go_version; then
+		return
+	fi
 	go_version="$(cat "${GITHUB_WORKSPACE}/GO_VERSION")"
 	wget -O go.tar.gz "https://dl.google.com/go/go${go_version}.linux-amd64.tar.gz"
-	tar -C /usr/local -xzf go.tar.gz
+	echo "Extracting Go ${go_version}..."
+	<<-'SUDO_COMMANDS' sudo sh
+		set -o errexit
+		go_dir="$(
+			dirname "$(
+				dirname "$(
+					realpath "$(
+						which go
+						)")")")"
+		mv "$go_dir" "${go_dir}.unused"
+		tar -C /usr/local -xzf go.tar.gz
+	SUDO_COMMANDS
 	rm go.tar.gz
 	export PATH="${PATH}:${GOROOT}/bin"
 	go version
 }
+
+GOROOT=/usr/local/go
+export GOROOT PATH="${PATH}:${GOROOT}/bin"
 download_go
 
 if [ -z "$INPUT_DIR" ]; then
 	# There's a bug in "defaults" for inputs
 	INPUT_DIR="./lib/..."
 fi
 
-export GOPATH="$(mktemp -d)"
-srcdir="$GOPATH/src/github.com/apache"
-mkdir -p "$srcdir"
-ln -s "$PWD" "$srcdir/trafficcontrol"
-cd "$srcdir/trafficcontrol"
+GOROOT=/usr/local/go
+export GOROOT PATH="${PATH}:${GOROOT}/bin"
```

Comment:
This is a bit nit-picky, but I feel like this would be easier as
```bash
export GOROOT="/usr/local/go"
export PATH="${PATH}:${GOROOT}/bin"
```
since you're using two lines already anyway

Reply:
These additions are gone as of be592797c5.